quic/s2n-quic-core/src/datagram/default.rs

```diff
@@ -21,20 +166,34 @@ impl Sender for DefaultSender {
         if packet.has_pending_streams() && !packet.datagrams_prioritized() {
             return;
         }
-
+        DefaultSender::record_capacity_stats(self, packet.remaining_capacity());
```

Comment:
you can just call this directly

```suggestion
        self.record_capacity_stats(packet.remaining_capacity());
```